frontend/src/app-components/dialogs/confirm/ConfirmDialogBody.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright © 2025 Hexastack. All rights reserved.
+ *
+ * Licensed under the GNU Affero General Public License v3.0 (AGPLv3) with the following additional terms:
+ * 1. The name "Hexabot" is a trademark of Hexastack. You may not use this name in derivative works without express written permission.
+ * 2. All derivative works must include clear attribution to the original creator and software, Hexastack and Hexabot, in a prominent location (e.g., in the software's "About" section, documentation, and README file).
+ */
+
+import ErrorIcon from "@mui/icons-material/Error";
+import { Grid, Typography } from "@mui/material";
+
+import { useTranslate } from "@/hooks/useTranslate";
+
+export const ConfirmDialogBody = () => {
+  const { t } = useTranslate();
+
+  return (
+    <Grid container gap={1}>
+      <Grid item height="28px">
+        <ErrorIcon sx={{ fontSize: "28px" }} color="error" />
+      </Grid>
+      <Grid item alignSelf="center">
+        <Typography>{t("message.item_delete_confirm")}</Typography>
```

Comment:
We are not customizing the message depending if it's singular or plural, right?

Reply:
I will include the singular or plural delete updates in this PR